pkgs/applications/window-managers/sway/osd.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, gtk3
+, gtk-layer-shell
+, libpulseaudio
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "swayosd";
+  version = "unstable-20230509";
```

```suggestion
  version = "unstable-2023-05-09";
```

Or maybe `unstable-2023-05-08`.